### PR TITLE
Order Styles Files

### PR DESCRIPTION
**Pull request by aannyyaa**
Make styles files more modular

### CODE REVIEW

File: src/styles/_footer.scss [hunk 1]
```diff
@@ -0,0 +1,131 @@
+/* || MOBILE */
+
+@media (min-width: units('mobile')) {
+  /* || FOOTER  */
```
Comment (saifulislamdev) [inline src/styles/_footer.scss]: extra

File: src/styles/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    a {
+      border: none;
+    }
+  }
+}
+
+/* || DESKTOP */
+
+@media (min-width: units('desktop')) {
+  /* || FOOTER */
```
Comment (saifulislamdev) [inline src/styles/_footer.scss]: extra

File: src/styles/_header.scss [hunk 3]
```diff
@@ -0,0 +1,284 @@
+/* || MOBILE */
+
+@media (min-width: units('mobile')) {
+  
+  /* || HEADER */
```
Comment (saifulislamdev) [inline src/styles/_header.scss]: extra

File: src/styles/_header.scss [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+      }
+    }
+  }
+}
+
+/* || DESKTOP */
+
+@media (min-width: units('desktop')) {
+
+  /* || HEADER */
```
Comment (saifulislamdev) [inline src/styles/_header.scss]: extra

File: src/styles/_footer.scss (hunk 1 above)
Comment (aannyyaa) [reply src/styles/_footer.scss]: I removed the comments you highlighted

**Review by saifulislamdev — APPROVED** [review]

**Pull request merged**